package/Dockerfile

```
@@ -53,7 +53,7 @@ ENV CATTLE_SYSTEM_AGENT_UPGRADE_IMAGE rancher/system-agent:${CATTLE_SYSTEM_AGENT
 ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 101.0.0+up0.3.3
 
 # System charts minimal version
-ENV CATTLE_FLEET_MIN_VERSION=101.0.0+up0.5.0
+ENV CATTLE_FLEET_MIN_VERSION=101.0.1+up0.5.0
```

Update version from https://github.com/rancher/charts/pull/2347/files#diff-c1c585d7e4221b4d16ba06e707429655b4a59f0ebc07df034ef903bd6d353afeR15
```suggestion
ENV CATTLE_FLEET_MIN_VERSION=101.0.1+up0.5.1
```

---

Will happen here: https://github.com/rancher/rancher/pull/40263/files#diff-c55d1ee74f32cd4b9eeeceaec981a180f52e761acb158a5b0b0e088c7725c9caR56